static/js/stream_edit.js

```diff
@@ -219,6 +227,12 @@ function show_subscription_settings(sub_row) {
             return item.email;
         },
     });
+    const realm_user_count = people.get_active_human_persons().length;
+    const human_subscriber_count = exports.get_human_subscribers_count(sub.subscribers);
+    if (realm_user_count - human_subscriber_count === 0) {
```

Comment:
`realm_user_count` can just be `people.get_realm_count()` then there is no need for the extra variable. Per the function definition:
```js
exports.get_realm_count = function () {
    // This returns the number of active people in our realm.  It should
    // exclude bots and deactivated users.
    return active_user_dict.size;
};
```

Reply:
I checked this and this function includes bots too. That's why I wrote `people.get_active_human_persons().length`.
![Screenshot from 2020-03-15 18-59-45](https://user-images.githubusercontent.com/42106909/76702361-34f66000-66ef-11ea-9adb-d463dae034c3.png)

Reply:
Huh, @showell is this the right behavior and the comment is just out of date?

Reply:
@whoodes When I pointed this out in the chat, someone started a discussion and now this function is yet to be decided upon. https://github.com/zulip/zulip/pull/14216